[2023-12-08 11:07:05] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:20130)的加密handshake与协议handshake!

[2023-12-08 11:07:05] [info] server.go:602 dpchain/p2p/server.(*Server).listenLoop 接收到来自节点 127.0.0.1:52990的p2p连接请求

[2023-12-08 11:07:05] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:52990)的加密handshake与协议handshake!

[2023-12-08 11:07:05] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:52990)的加密handshake!

[2023-12-08 11:07:05] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:52990 

[2023-12-08 11:07:05] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:20130)的加密handshake!

[2023-12-08 11:07:05] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:20130 

[2023-12-08 11:07:05] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn a226effe746e5dd9 127.0.0.1:52990 failed checkpoint posthandshake: Connected to self
[2023-12-08 11:07:05] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn a226effe746e5dd9 127.0.0.1:20130 failed checkpoint posthandshake: Connected to self
[2023-12-08 11:07:37] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:20130)的加密handshake与协议handshake!

[2023-12-08 11:07:37] [info] server.go:602 dpchain/p2p/server.(*Server).listenLoop 接收到来自节点 127.0.0.1:53183的p2p连接请求

[2023-12-08 11:07:37] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:53183)的加密handshake与协议handshake!

[2023-12-08 11:07:37] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:53183)的加密handshake!

[2023-12-08 11:07:37] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:53183 

[2023-12-08 11:07:37] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn a226effe746e5dd9 127.0.0.1:53183 failed checkpoint posthandshake: Connected to self
[2023-12-08 11:07:37] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:20130)的加密handshake!

[2023-12-08 11:07:37] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:20130 

[2023-12-08 11:07:37] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn a226effe746e5dd9 127.0.0.1:20130 failed checkpoint posthandshake: Connected to self
[2023-12-08 11:07:49] [warning] peer.go:208 dpchain/p2p/server.(*Peer).pingLoop heart beate, ping协程退出

[2023-12-08 11:07:49] [info] separator.go:119 dpchain/core/separator.(*Separator).handlePeer.func1 与节点(f4124d176a5f5f8caa73746bcbf394e9a0ff64984e4aac4fdd029140eaca9f4219a36e11344ed4b6f7b6799c91100d7c0da3373f6dc2cd0871194834d6870d00)断开separator handshake!

[2023-12-08 11:07:49] [info] server.go:717 dpchain/p2p/server.(*Server).runPeer Remove Protocal peer (NodeID:6634313234643137366135663566386361613733373436626362663339346539613066663634393834653461616334666464303239313430656163613966343231396133366531313334346564346236663762363739396339313130306437633064613333373366366463326364303837313139343833346436383730643030) (srcIPAddr:127.0.0.1:55877) (destIPAddr:127.0.0.1:20132) ,err:Network error

[2023-12-08 11:07:49] [warning] peer.go:319 dpchain/p2p/server.(*Peer).startProtocols.func1 协议handshake, 协议Run函数错误,err:EOF

[2023-12-08 11:07:49] [error] dial.go:217 dpchain/p2p/server.(*dialTask).Do dial error: dial tcp 127.0.0.1:20132: connect: connection refused
[2023-12-08 11:07:49] [info] whisper.go:221 dpchain/p2p/whisper.(*Whisper).handlePeer.func1 与节点 (6634313234643137366135663566386361613733373436626362663339346539613066663634393834653461616334666464303239313430656163613966343231396133366531313334346564346236663762363739396339313130306437633064613333373366366463326364303837313139343833346436383730643030) 断开 whisper handshake!

[2023-12-08 11:07:49] [warning] peer.go:319 dpchain/p2p/server.(*Peer).startProtocols.func1 协议handshake, 协议Run函数错误,err:EOF

[2023-12-08 11:07:49] [info] separator.go:119 dpchain/core/separator.(*Separator).handlePeer.func1 与节点(31d981866fcafb83f4a92dc5636564f161a9940e063c96b0d8109174459cea4a89407ba20a7725aee1ab9e573f401496c3c498acd1315ec8ebac19999cb14491)断开separator handshake!

[2023-12-08 11:07:49] [warning] peer.go:319 dpchain/p2p/server.(*Peer).startProtocols.func1 协议handshake, 协议Run函数错误,err:EOF

[2023-12-08 11:07:49] [info] whisper.go:221 dpchain/p2p/whisper.(*Whisper).handlePeer.func1 与节点 (3331643938313836366663616662383366346139326463353633363536346631363161393934306530363363393662306438313039313734343539636561346138393430376261323061373732356165653161623965353733663430313439366333633439386163643133313565633865626163313939393963623134343931) 断开 whisper handshake!

[2023-12-08 11:07:49] [warning] peer.go:319 dpchain/p2p/server.(*Peer).startProtocols.func1 协议handshake, 协议Run函数错误,err:EOF

[2023-12-08 11:07:49] [warning] peer.go:208 dpchain/p2p/server.(*Peer).pingLoop heart beate, ping协程退出

[2023-12-08 11:07:49] [info] server.go:717 dpchain/p2p/server.(*Server).runPeer Remove Protocal peer (NodeID:3331643938313836366663616662383366346139326463353633363536346631363161393934306530363363393662306438313039313734343539636561346138393430376261323061373732356165653161623965353733663430313439366333633439386163643133313565633865626163313939393963623134343931) (srcIPAddr:127.0.0.1:55876) (destIPAddr:127.0.0.1:20131) ,err:Network error

[2023-12-08 11:07:49] [error] dial.go:217 dpchain/p2p/server.(*dialTask).Do dial error: dial tcp 127.0.0.1:20131: connect: connection refused
